main.py

```diff
@@ -2,6 +2,80 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
+
+def create(data: list) -> None:
```

Comment:
Is ok, but a bit overcomplicated - you could just run two for loops - one for creating Genres and one for creating actors. But I see what you did and it's great that you parametrized input 👏